cmd/server-manager/views/pages/server/logs.html

```diff
@@ -7,6 +7,8 @@ <h2>Assetto Corsa Server</h2>
     <div class="card card-body bg-light card-logs">
         <pre id="server-logs"></pre>
     </div>
+    <br>
+    <a class="btn btn-primary" href="/api/log-download/ServerLog">Download Server Log</a>
```

Comment:
could these be `server`, `manager`, `plugins` instead of `ServerLog` etc? just a small naming thing - i'm not a huge fan of capitals in URLs :)